fuzz/fuzz_targets/fuzz.rs

```diff
@@ -319,6 +328,56 @@ impl<const D: usize> Workload<D> {
         traverse
     }
 
+    /// Compares normal and flat-BVH nearest_from query.
+    ///
+    /// # Panics
+    /// If `assert_agreement` is true, panics if the results differ in an
+    /// unexpected way.
+    fn fuzz_nearest_from<'a>(
+        &'a self,
+        bvh: &'a Bvh<Float, D>,
+        flat_bvh: &'a FlatBvh<Float, D>,
+        query_point: Point<Float, D>,
+    ) {
+        let best = bvh
+            .nearest_from(query_point, &self.shapes)
+            .map(|(s, d)| (ByPtr(s), d));
+        let flat_best = flat_bvh
+            .nearest_from(query_point, &self.shapes)
+            .map(|(s, d)| (ByPtr(s), d));
```

Comment:
I think the `ByPtr` wrapper isn't necessary here (or later in this function) because, as you say, you don't want to compare the shapes.